lib/Google/RestApi.pm

```diff
@@ -32,32 +34,51 @@ sub new {
   my $class = shift;
 
   state $check = compile_named(
-    config_file => Str, { optional => 1 },
-    _extra_     => slurpy Any,
+    config_file          => Str, { optional => 1 },
+    service_account_file => Str, { optional => 1 },
```

Comment:
config_file should be used to contain all possible keys, so service_account_file should not be included here. In other words, service_account_file could be specified in config_file, or just passed as part of the config hash.

Reply:
agreed, I just changed that.